woocommerce/woocommerce-gutenberg-products-block
# Add `SortSelect` to Storybook

## What

Adds `SortSelect` to Storybook.

Also adds the `disabled` prop onto the `select` if `readOnly` is true. Previously this was defined in the props for the component but was never used.

## Why

To illustrate the component's use and provide some level of documentation.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Open Storybook, ensure the `Checkout Components -> SortSelect` renders and the controls work.
2. Set up some reviews on your products (Go to the product on the front end then the Reviews tab)
3. Add the All Reviews and All Products blocks to a page.
4. Visit that page and ensure the select box to change the ordering of reviews is visible. Interact with it and ensure the order of the reviews changes.
5. On the All Products block, ensure the select box to change the ordering of products is visible. Interact with it and ensure the order of products changes.


## Screenshots or screencast



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Skipping
